packages/material-ui/package.json

```diff
@@ -3,8 +3,8 @@
   "version": "5.0.0-alpha.22",
   "private": false,
   "author": "Material-UI Team",
+  "description": "Quickly build beautiful React apps. Simple, customizable, and accessible React components. Build your own design system, or start with Material Design.",
```

Comment:
```suggestion
  "description": "Quickly build beautiful React apps. Simple, customizable, and accessible React components. Build your own design system, or start with Material Design.",
```

- Remove `simple`
  This is subjective and we already avoid it in the docs for that reason
- Remove `accessible`
  Misleading term that has been trending to be just a buzzword. It has no longer any meaning in the ecosystem and we're big enough to not fall into the same trap.

Reply:
> simple
 
From what I understand, "simple" is defined by opposition to complex. "complex" is defined as "consisting of many different and connected parts". So the subjectively is either the component connect more parts than they need to. How about?

```suggestion
  "description": "Quickly build beautiful React apps. Painless, customizable, and accessible React components. Build your own design system, or start with Material Design.",
```

> accessible

The main incentive to include was to mention that it's something we dedicate time to. We might not prioritize it as much as you wish we did, but we still do account for accessibility (relatively to developers' demand), on the published components, a lot more than, say, Ant Design.

Reply:
> The main incentive to include was to mention that it's something we dedicate time to.

Your intention is fine. But the result is just incorrect. Just because you dedicate time to make something X does not mean it is X. 

> From what I understand, "simple" is defined by opposition to complex.

It's tiring. Why do we need to discuss this again all of the sudden? Do you wish me to collect all the previous changes where we removed "simple" from the docs?

Reply:
> It's tiring. Why do we need to discuss this again all of the sudden? Do you wish me to collect all the previous changes where we removed "simple" from the docs?

I was detailing my thoughts process on why I agree with your comment. Simple is subjective as relative to one consider unrelated parts mixed together. It can be an objective but it's up to each to interpret if it's true or not.

Reply:
> Your intention is fine. But the result is just incorrect. Just because you dedicate time to make something X does not mean it is X.

If we want the description to focus on the why, then using simple + accessible is great, if we want the description to focus on the what, then we need to find something else. @mbrookes What do you think?

Reply:
@mbrookes Your proposal to replace apps with UI to remove the potential confusion seems to fly:

<img width="437" alt="Capture d’écran 2021-01-11 à 21 31 50" src="https://user-images.githubusercontent.com/3165635/104234956-84fb9b80-5454-11eb-8077-513e6e704552.png">

```suggestion
  "description": "Quickly build beautiful React UI. Simple, customizable, and accessible React components. Build your own design system, or start with Material Design.",
```

Reply:
> Or should it be UI instead of UIs

"Quickly build  a beautiful React UI." if you prefer the singular.

> Build your own design system, or start with Material Design.

"Follow your own design system"?

Reply:
```suggestion
  "description": "Quickly build beautiful React UIs. Simple, customizable, and accessible React components. Follow your own design system, or start with Material Design.",
```

Reply:
> @mbrookes What do you think?

As a marketing tagline, subjective terms are almost inevitable. If, relative to alternative solutions, they stand up, I'm okay with that. Is accessibility perfect? No, and probably never will be – we can always improve. Is it better than the next most popular alternative? Yes by several orders of magnitude.

If alternatively we reject all relative terms, we're left with "Sixty-four React components", which while factual, is hardly compelling.. 🙂

Reply:
I have replaced simple with painless. I also think that we should focus on the why.

Reply:
"painless" -> "developer friendly"?

Reply:
I find the proposal not strong enough and too long to read.

Reply:
I find the "negated negative" of "painless" still has negative connotations.

Reply:
True 🤔

Reply:
I got the idea from

> React makes it painless to create interactive UIs. Design simple views for each state in your application, and React will efficiently update and render just the right components when your data changes.

https://reactjs.org/

Reply:
Picking a few words from third party reviews describing the components: lightweight, consistent, clean, configurable, useful, flexible, rock solid. (Pick one. 😄 )

I quite liked this description from one of them:
"build a beautiful, consistent, and light interface quickly without sacrificing accessibility and performance"

Reply:
Not sure, I think that "Quickly build beautiful React UIs" and "Follow your own design system, or start with Material Design." are already action-driven. I think that we need to have something destructive and bold (even if overeaching)

Reply:
"Material-UI is a simple and customizable component library to build faster, beautiful, and more accessible React applications. Follow your own design system, or start with Material Design."

?

<img width="627" alt="Capture d’écran 2021-01-14 à 13 59 07" src="https://user-images.githubusercontent.com/3165635/104593923-b486ef80-5670-11eb-8401-b6df1fed12b3.png">

Reply:
Ship it, Danno!

Reply:
Updated